### PR TITLE
Adding the support to retrieve and load semantic masks from konva canvas

### PR DESCRIPTION
**Pull request by pritamDas9191**
getLoadMask

### CODE REVIEW

**Review by code-vicar — DISMISSED** [review]

**Review by code-vicar — DISMISSED** [review]

**Review by code-vicar — APPROVED** [review]

File: src/canvastools/ts/CanvasTools/Region/Mask/MaskManager.ts [hunk 1]
```diff
@@ -115,6 +117,200 @@ export class MasksManager {
...
+            
+            const ctx = canvas?.getContext("2d");
+            const data: ImageData = ctx.getImageData(0, 0, currentDimensions.width, currentDimensions.height);
+
+            this.konvaStage.width(width).height(height).scaleX(scaleX).scaleY(scaleY);
+
+            // create an edge pixel matrix
+            const edgeArray = this.getEdgePixelArray(data.data);
+
+            // remove antialiasing/smoothening ?
```
Comment (HarshUpadhyay-MSFT) [inline src/canvastools/ts/CanvasTools/Region/Mask/MaskManager.ts:148]: nit: this comment has a question mark so just wanted to double check if we have finalized whether to remove AA from data

File: tslint.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "extends": ["tslint:recommended", "tslint-config-prettier"],
     "linterOptions": {
         "include": ["src/**/*.ts"],
-        "exclude": ["**/*.d.ts"]
+        "exclude": ["**/*.d.ts", "**/*.spec.ts"]
```
Comment (HarshUpadhyay-MSFT) [inline tslint.json:6]: nit: why is it necessary to exclude spec files?

**Review by HarshUpadhyay-MSFT — COMMENTED** [review]
Minor questions

File: tslint.json (hunk 2 above)
Comment (pritamDas9191) [reply tslint.json:6]: yes removed

**Pull request merged**